content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Common Issues
+layout: faq
+---
+
+The topics below are the most common issues that we identify when
+validating apps to run on our platform.  We suggest you work through
+each of these before submitting an app for validation.
+
+# Common Issues
+
+* TOC
+{:toc}
+
+## API Usage
+
+### Avoid Multiple Resource Calls
+
+Ensure that your app is efficient in the way it makes API calls.
+Duplicate API calls should be removed to avoid additional performance
+load on the system and improve application load time. As part of the
+technical review process, we will review logs to see if calls are
+identical each time and if they are retrieving the same data set.
+
+
+### Implement Paging
+
+To be compatible with a wide variety of FHIR services and for future-
+proofing, we require that you implement a paging data retrieval scheme
+as defined by the [HL7 FHIR specification](https://www.hl7.org/fhirhttp.html#paging).
```

Comment:
We need to be careful with all these links. This will always go to the latest spec, which may change the way paging is defined. For FHIR, all our links are to the versioned FHIR doc. EG: https://www.hl7.org/fhir/dstu2/http.html#paging